conf/deployment/aws/upi_1az_rhel7_3m_3w.yaml

```diff
@@ -0,0 +1,15 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+ENV_DATA:
+  platform: 'aws'
+  deployment_type: 'upi'
+  worker_replicas: 3
+  master_replicas: 3
+  rhel_workers: true
+  rhcos_ami: 'ami-06c85f9d106577272'
+  rhel_worker_ami: 'ami-073251e3a0e36cebb' # RHEL7.7 as of now
+  rhel_worker_instance_type: 'm5.4xlarge'
+  root_disk_size: 50
```

Comment:
Coady, according Jira requirements root disk size should be 120GB.

Reply:
https://jira.coreos.com/browse/KNIP-658  @vavuthu  BTW do we following this also for VmWare? Thanks

Reply:
Its 60GB. I will submit PR for changing to 120 GB

Reply:
Done

Reply:
Done